### PR TITLE
Move blockquotes style to .answers {}

### PR DESCRIPTION
**Pull request by rlr**
tiny r?

### CODE REVIEW

**Comment by rlr** [comment]
this just moves the style block for `blockquotes` to the right section.

**Comment by mythmon** [comment]
Oops, thanks. r+

File: kitsune/sumo/static/less/questions.less [hunk 1]
```diff
@@ -218,6 +218,13 @@ h2 {
       > p {
         white-space: pre-wrap;
       }
+
+      blockquote {
+        margin: 1.5em 0em;
```
Comment (rehandalal) [inline kitsune/sumo/static/less/questions.less]: m > b

**Comment by rlr** [comment]
fbb1ffe9286df7559ff70203c522be3319e21563 Move blockquotes style to .answers {}

**Pull request closed without merging**